airbyte-integrations/connectors/source-airtable/source_airtable/source.py

```diff
@@ -53,7 +53,7 @@ def discover(self, logger: AirbyteLogger, config) -> AirbyteCatalog:
                     {
                         "stream_path": f"{base_id}/{table.get('id')}",
                         "stream": SchemaHelpers.get_airbyte_stream(
-                            f"{base_name}/{SchemaHelpers.clean_name(table.get('name'))}",
+                            f"{base_name}/{SchemaHelpers.clean_name(table.get('name'))}/{table.get('id')}",
```

Comment:
Why use the name at all, if the ID is the unique thing?

Reply:
@evantahler , Customers will have to refresh source schema (with resetting).
ID is unique, but it is better to show "normal" name as well for user, e.g. [here](https://github.com/airbytehq/airbyte/pull/22704/files#diff-dfd0d62d60cf384182b1fd813d87208d94cca591ba4a12f8a5c68697a1356523R75) just using ID is not enough to distinguish tables.

Reply:
version bumped.